<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2e2e2e; /* Dark gray for the background */
    border: 2px solid #444444; /* Slightly lighter gray for the border */
    border-radius: 10px;
    box-shadow: 0px 2px 5px rgba(0, 0, 0, 0.3); /* Subtle shadow for depth */</string>
  </property>
  <widget class="QLabel" name="lbl_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #dddddd;</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_artist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #dddddd;</string>
   </property>
   <property name="text">
    <string>Artist: </string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_album">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #dddddd;</string>
   </property>
   <property name="text">
    <string>Album:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_playcount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #dddddd;</string>
   </property>
   <property name="text">
    <string>Played:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_play">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: #4caf50; /* Green button for a clean look */
    color: white;
    border: none;
    border-radius: 5px;
    padding: 5px 15px;
    margin-top: 5px;</string>
   </property>
   <property name="text">
    <string>Nghe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: #4caf50; /* Green button for a clean look */
    color: white;
    border: none;
    border-radius: 5px;
    padding: 5px 15px;
    margin-top: 5px;</string>
   </property>
   <property name="text">
    <string>ThÃªm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
